src/main/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolver.scala

```diff
@@ -32,9 +32,15 @@ private[marathon] class InstanceUpdateOpResolver(
   def resolve(op: InstanceUpdateOperation)(implicit ec: ExecutionContext): Future[InstanceUpdateEffect] = {
     op match {
       case op: Schedule =>
-        createInstance(op.instanceId) {
-          // TODO(karsten): Create events
-          InstanceUpdateEffect.Update(op.instance, oldState = None, Seq.empty)
+        // TODO(karsten): Create events
+        createOrUpdateInstance(op.instanceId, InstanceUpdateEffect.Update(op.instance, oldState = None, Seq.empty)) { i =>
+          (i.agentInfo, i.reservation) match {
+            case (Some(agentInfo), Some(reservation)) =>
+              // we already have suspended instance with reservation, let's schedule that one
```

Comment:
Ni: `have a suspended instance` It might be confused with `we have suspended the instance`